tuxemon/event/actions/set_monster_status.py

```diff
@@ -65,7 +65,7 @@ def set_status(monster: Monster, value: Optional[str]) -> None:
             # TODO: own class for status effect
             # TODO: handle invalid statues
             status = Technique(value)
-            monster.apply_status(status)
+            monster.apply_status(monster, status)
```

Comment:
Why?

Reply:
Because it's the same function in monster.py, if not set_monster_status action wouldn't work.

Reply:
But it's a method on monster, why do you need to pass monster in again as an argument?

Reply:
You're right, it's a duplicate, I'm going to get rid of.